libyul/AssemblyStack.cpp

```diff
@@ -232,6 +233,62 @@ MachineAssemblyObject AssemblyStack::assemble(Machine _machine) const
 	return MachineAssemblyObject();
 }
 
+std::string AssemblyStack::computeSourceMapping(evmasm::AssemblyItems const& _items) const
```

Comment:
Can you unify this with the other function of the same name and move them both to a static function of AssemblyItem?

Reply:
Done